src/foam/nanos/u2/navigation/TopNavigation.js

```diff
@@ -132,7 +137,7 @@ foam.CLASS({
         .start()
           .addClass('logged-in-container')
           .tag({ class: 'foam.nanos.u2.navigation.BusinessLogoView' })
-          .start({ class: 'foam.nanos.menu.MenuBar' })
+          .start(this.MenuBar, { menuName$: this.group$.dot('rootMenu') })
```

Comment:
Do we need to bind slots? Is the menu going to redraw if the group or group.rootMenu changes? And are they ever going to change?

Reply:
Just felt more robust this way in case group ever does change but the reality is that the MenuBar doesn't bind to the menuName so even if it does change, this won't update.